packages/editor-ui/src/components/CanvasChat/CanvasChat.test.ts

```diff
@@ -230,28 +230,28 @@ describe('CanvasChat', () => {
 			// Verify workflow execution
 			expect(workflowsStore.runWorkflow).toHaveBeenCalledWith(
 				expect.objectContaining({
-					runData: {
-						'When chat message received': [
-							{
-								data: {
-									main: [
-										[
-											{
-												json: {
-													action: 'sendMessage',
-													chatInput: 'Hello AI!',
-													sessionId: expect.any(String),
-												},
+					runData: undefined,
+					triggerToStartFrom: {
+						name: 'When chat message received',
+						data: {
+							data: {
+								main: [
+									[
+										{
+											json: {
+												action: 'sendMessage',
+												chatInput: 'Hello AI!',
+												sessionId: expect.any(String),
 											},
-										],
+										},
 									],
-								},
-								executionStatus: 'success',
-								executionTime: 0,
-								source: [null],
-								startTime: expect.any(Number),
+								],
 							},
-						],
```

Comment:
The `runData` property is not used by the BE anymore to execute chat trigger workflows since https://github.com/n8n-io/n8n/pull/11952

The BE now uses `triggerToStartFrom`, so I changed the test to assert that property instead.